### ISSUE TITLE
Linux APT chat shortcuts not functioning in proton

### ISSUE THREAD

**Issue by chris-stabb** [SnosMe/awakened-poe-trade#390]
I'm playing POE through steamplay via Proton 6.3.3 on Manjaro Linux. The item shortkeys for pricing and maps work correctly. 

However the shortkeys for the chat functions will work one time, and then all subsequent key presses will just paste whatever is on the clipboard into the chat window. I set the /hideout command to Shift + 2, and /exit to Shift + 1. These work a single time, and both show the same behavior of pasting anything on my clipboard into chat.

**Comment by SnosMe** [comment]
Unfortunately, sending keystrokes in Linux causes X11 to lag, and you can observe strange behavior.

Try disable this

![image](https://user-images.githubusercontent.com/4292308/118114322-19204400-b3f0-11eb-9509-71cb1a5aa3df.png)

and change to "Hook" mode

![image](https://user-images.githubusercontent.com/4292308/118114342-1f162500-b3f0-11eb-989e-8eff45f8f183.png)

**Comment by tinycrops** [comment]
I investigated a related Linux/Proton chat-macro failure where a configured command such as `/menagerie` intermittently became a literal `v` in chat.

The failure depends on in-game UI state: it happens much more often when inventory or another panel is open. Delaying/holding the synthetic Ctrl+V chord itself did not fix it. The effective change was to wait after the initial Enter event, before sending Ctrl+V.

This suggests PoE's chat edit control is focused asynchronously under Wine/Proton. When APT sends Ctrl+V immediately after Enter, the game can finish its focus transition after the injected modifier state has passed and handle only V.

Tested change:

- On Linux, wait 150 ms after `Enter` opens chat before select/paste/send.
- Extend the Linux clipboard restoration window from 120 ms to 250 ms so the command is still present when the delayed paste occurs.
- Leave Windows and macOS timing unchanged.

A 100 ms version improved real-game F6/F7 reliability by roughly 90% (about 49 successful activations out of 50). The remaining failure was with inventory open in another player's hideout, so the published branch uses 150 ms.

Patch/branch:
https://github.com/tinycrops/awakened-poe-trade/compare/master...tinycrops:awakened-poe-trade:agent/fix-linux-chat-paste-race

Commit:
https://github.com/tinycrops/awakened-poe-trade/commit/4800e7e5c305ed98ed22ecf92021a282d408b258

Validation: `npm run build`, production Linux AppImage build, and repeated in-game testing with inventory panels open and closed.
